tests/ui_tests/app/page/admin/page.py

```diff
@@ -489,3 +510,83 @@ def click_delete_button(self):
         self.wait_element_visible(DeleteDialog.body)
         self.find_and_click(DeleteDialog.yes)
         self.wait_element_hide(DeleteDialog.body)
+
+
+class AdminPoliciesPage(GeneralAdminPage):
+    """Admin Policy Page class"""
+
+    MENU_SUFFIX = 'policies'
+    MAIN_ELEMENTS = [
+        AdminIntroLocators.intro_title,
+        AdminIntroLocators.intro_text,
+        AdminGroupsLocators.create_group_btn,
+        AdminGroupsLocators.delete_btn,
+        CommonTable.header,
+    ]
+
+    def click_create_policy_btn(self):
```

Comment:
I suggest naming it like `open_create_policy_popup` and clear it in docs that it is opened by clicking button

Reply:
changed